frontend/src/pages/Admin/Users/UserRow/EditUserModal/index.jsx

```diff
@@ -53,10 +59,14 @@ export default function EditUserModal({ currentUser, user, closeModal }) {
                   className="bg-zinc-900 placeholder:text-white/20 border-gray-500 text-white text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5"
                   placeholder="User's username"
                   minLength={2}
-                  defaultValue={user.username}
+                  value={username}
```

Comment:
If we edit only a users password - does the new validation block a previously working username from being updated?